### PR TITLE
DISTMYSQL-195 Release notes PDPXC 8.0.28

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
LGTM

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Release Notes
 
+* [Percona Distribution for MySQL 8.0.28 using Percona XtraDB Cluster](release-notes-pxc-v8.0.28.md)
```
Comment (LenzGr) [inline docs/release-notes.md:3]: These links still miss the release dates - are you going to add these as a separate PR?

Comment (nastena1606) [reply docs/release-notes.md:3]: No, I will update this one once the release dates are defined

File: docs/release-notes-pxc-v8.0.28.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.28 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-28.html).
+
+The following is the list of components supplied with the *Percona XtraDB Cluster*-based deployment variant of Percona Distribution for MySQL:
+
+| Component           | Version   | Description                                |
+| ------------------- | --------- | -------------------------------------------|
+| Percona XtraBackup  | 8.0.28    | An open-source hot backup utility for MySQL-based servers that doesn’t lock your database during the backup.|
+| HAProxy             | 2.5.6    | A high-availability and load-balancing solution for Percona XtraDB Cluster. This is a default proxy.|
+| ProxySQL            | 2.3.2     | A high performance, high-availability, protocol-aware proxy for MySQL.          |
```
Comment (LenzGr) [inline docs/release-notes-pxc-v8.0.28.md]: I wonder if we could add links to the component's respective project home pages and release notes, where applicable - for ProxySQL, the name should link to https://docs.percona.com/proxysql/ and the version number should link to https://docs.percona.com/proxysql/release-notes-2.3.2-1.html
This way, the reader can easily learn about changes in the other components that we include (not just the core database)
What do you think?

Comment (nastena1606) [reply docs/release-notes-pxc-v8.0.28.md]: Having multiple links is an overkill IMO. We do have the links to the components on index page. Yet to link the version number to the release notes is a great idea, thank you. However, should we always provide such a link? Not all components may be updated with the distro release. Do we duplicate the URL for non-updated components from the previous release?

Comment (LenzGr) [reply docs/release-notes-pxc-v8.0.28.md]: > Having multiple links is an overkill IMO. We do have the links to the components on index page.

OK, makes sense to leave them out here then.

> Yet to link the version number to the release notes is a great idea, thank you. However, should we always provide such a link? Not all components may be updated with the distro release. Do we duplicate the URL for non-updated components from the previous release?

Yes, I think that is fine. The distribution release notes should be useful for both users updating from a previous version as well as new users that install our distribution for the first time.

Comment (nastena1606) [reply docs/release-notes-pxc-v8.0.28.md]: Updated

**Pull request merged**